spec/rails_helper.rb

```diff
@@ -28,6 +28,12 @@
 
   config.include(ValidationHelpers, type: :model)
 
+  # Adding support for url_helper
+  config.include Rails.application.routes.url_helpers
+
+  # Adding support for url_helper
+  config.include Rails.application.routes.url_helpers
```

Comment:
Did you actually mean to include this twice? Do we only want to maybe include it for controller and request specs and not for unit specs?